design/ironic-debuggability-improvement.md

```diff
@@ -0,0 +1,108 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# ironic-debuggability-improvement
+
+## Status
+
+provisional
+
+## Summary
+
+Ironic and Ironic-inspector logs are currently hidden inside their respective
```

Comment:
To be clear, this is about ramdisk logs, not the regular service logs.

Reply:
Good point. Added "ramdisk" to clarify.